cmd/tetragon/main.go

```diff
@@ -133,6 +135,13 @@ func tetragonExecute() error {
 		log.Fatal(err)
 	}
 
+	option.Config.TracingPolicyDir = filepath.Join("/", option.Config.TracingPolicyDir)
+	tracingPolicyDir, err := filepath.Abs(option.Config.TracingPolicyDir)
```

Comment:
if you want to clean up the file (which we now is absolute), just call `.Clean()`

Reply:
done